test/pwd.js

```diff
@@ -23,6 +23,6 @@ var _pwd = shell.pwd();
 assert.equal(_pwd.code, 0);
 assert.ok(!_pwd.stderr);
 assert.equal(shell.error(), null);
-assert.equal(path.basename(_pwd), 'tmp');
+assert.equal(path.basename(_pwd.toString()), 'tmp');
```

Comment:
It fails without this. In node v6, I believe `basename()` requires a regular string. If you can get it to pass without this, feel free to change as necessary